_docs/use_functions.md

```diff
@@ -0,0 +1,122 @@
+---
+title: Using Functions
+category: Common Concepts
+---
+
+You've already encountered the `main` function, the entrypoint of any Glu program, as well as the `func` keyword which is needed to declare a function.
+
+Procedural languages, like Glu, structure programs as a series of functions that call one another.
+This is why functions are essential in Glu programming.
+
+```glu
+func main() {
+    std::print("Hello World!");
+}
+```
+
+## Declare a Function
+
+To declare a function, four key components are required:
+- The `func` keyword
+- The function name
+- A list of any necessary parameters
+- The function potential returned value's type
+
+The body of the function is enclosed in curly brackets `{}`, which indicate to the compiler where the function begins and ends.
+
+In the example below, the function prototype of `my_func` specifies the function name and parameters: `param1` and `param2`, both of which are of type `Int`.
+
+![Representation of the differents key components required by a function declaration in Glu](/assets/img/function_explaining.png)
```

Comment:
The alt text should have the same information as the image, so it should include the code you’re talking about